pkg/cli/adbtunnel.go

```diff
@@ -138,25 +205,45 @@ func listADBTunnels(flags *listADBTunnelFlags, c *command, opts *subCommandOpts)
 			merr = multierror.Append(fmt.Errorf("Unable to contact tunnel agent: %w", err))
 			continue
 		}
-		status, err := sendStatusCmd(conn)
+		fwdr, err := sendStatusCmd(conn)
 		conn.Close()
 		if err != nil {
 			merr = multierror.Append(err)
 			continue
 		}
 
-		if flags.host != "" && status.Host != flags.host {
+		if zone != "" && zone != fwdr.Zone {
 			continue
 		}
 
-		line := fmt.Sprintf("%s/%s 127.0.0.1:%d %s",
-			status.Host, status.Device, status.Port, status.State)
-		if flags.longFormat {
-			line = fmt.Sprintf("%s: %s/%s", status.ServiceURL, status.Zone, line)
+		if host != "" && fwdr.Host != host {
+			continue
 		}
-		c.Printf("%s", line)
+
+		forwarders = append(forwarders, fwdr)
 	}
-	return merr
+	return forwarders, merr
+}
+
+func askUserToChoose(devices []ADBForwarderStatus, c *command) ([]ADBForwarderStatus, error) {
+	c.PrintErrln("Multiple ADB tunnels match:")
+	for i, d := range devices {
+		c.PrintErrf("%d) %s %s\n", i, d.Host, d.Device)
+	}
+	c.PrintErrf("%d) All\n", len(devices))
```

Comment:
These would look like this
```
Multiple matches:
1) Host1 cvd-1
2) Host2 cvd-1
3) Host2 cvd-2
4) All
Choose an option: _
```
Is it still confusing? I chose that because it's the same way acloud does it. It also makes more sense to me that all options are numbers, instead of some being numbers and others being letters.

Reply:
There is no default option btw, you have to consciously choose one

Reply:
would `All` be a common selection or vaguely used? if common, having it as the last option, with a different number every-time is not as good as having a constant value you can memorize once IMO.

Reply:
if you know you want to delete all without reading (memorized value) you can simply run `cvdremote adbtunnel close -y`

Reply:
there is no guarantee of consistency here, because devices can be listed in any order. Especially if you create more than one.

Reply:
Finally, reading a default value is almost impossible without using bufio (fmt.Scan* doesn't differentiate between IO error and no values read), but using bufio may read more than the selection string, which basically makes the global resource `stdin` in a bad state.